{"version": 2, "width": 151, "height": 50, "timestamp": 1719781779, "idle_time_limit": 1.0, "env": {"SHELL": "/bin/bash", "TERM": "xterm-256color"}}
[0.086825, "o", "\r\nThe default interactive shell is now zsh.\r\nTo update your account to use zsh, please run `chsh -s /bin/zsh`.\r\nFor more details, please visit https://support.apple.com/kb/HT208050.\r\n"]
[0.088802, "o", "\u001b[?1034h"]
[0.088853, "o", "bash-3.2$ "]
[2.40676, "o", "ssh user7@172.20.10.4"]
[2.811084, "o", "\r\n"]
[3.122094, "o", "\r"]
[3.122394, "o", "user7@172.20.10.4's password: "]
[5.560711, "o", "\r\n"]
[5.827018, "o", "Welcome to Ubuntu 22.04.4 LTS (GNU/Linux 6.5.0-35-generic x86_64)\r\n\r\n * Documentation:  https://help.ubuntu.com\r\n * Management:     https://landscape.canonical.com\r\n * Support:        https://ubuntu.com/pro\r\n\r\n1 device has a firmware upgrade available.\r\nRun `fwupdmgr get-upgrades` for more information.\r\n\r\n\r\nExpanded Security Maintenance for Applications is not enabled.\r\n\r\n31 updates can be applied immediately.\r\n26 of these updates are standard security updates.\r\nTo see these additional updates run: apt list --upgradable\r\n\r\n12 additional security updates can be applied with ESM Apps.\r\nLearn more about enabling ESM Apps service at https://ubuntu.com/esm\r\n\r\n\r\n1 device has a firmware upgrade available.\r\nRun `fwupdmgr get-upgrades` for more information.\r\n\r\nLast login: Mon Jul  1 00:06:35 2024 from 172.20.10.3\r\r\n"]
[5.910372, "o", "\u001b[?2004h\u001b[38;5;14m00:09:45 \u001b[38;5;224muser7\u001b[38;5;200m@\u001b[38;5;190mDubrovin02\u001b[38;5;10m$:~\r\r\n\u001b[m\u001b]12;#00ffff\u0007\u001b[38;5;14m$ \u001b[m"]
[8.796845, "o", "ba"]
[8.945807, "o", "s"]
[9.107379, "o", "h"]
[9.411164, "o", " "]
[9.717964, "o", "/"]
[10.583225, "o", "\b\u001b[K"]
[10.946887, "o", "D"]
[11.151979, "o", "o"]
[11.495327, "o", "\u0007"]
[12.18365, "o", "c"]
[12.492711, "o", "uments/"]
[13.320499, "o", "M"]
[13.726968, "o", "y"]
[13.741911, "o", "selfTrace/"]
[14.855134, "o", "install "]
[15.351775, "o", "\r\n\u001b[?2004l\r"]
[15.64417, "o", "Defaulting to user installation because normal site-packages is not writeable\r\n"]
[15.823419, "o", "\u001b[31mERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'\u001b[0m\u001b[31m\r\n\u001b[0mcp: cannot stat 'tracehost': No such file or directory\r\n\u001b[?2004h\u001b[38;5;14m00:09:55 \u001b[38;5;224muser7\u001b[38;5;200m@\u001b[38;5;190mDubrovin02\u001b[38;5;10m$:~\r\r\n\u001b[m\u001b]12;#00ffff\u0007\u001b[38;5;14m$ \u001b[m"]
[22.416283, "o", "c"]
[22.656947, "o", "d"]
[22.665626, "o", " "]
[23.397434, "o", "D"]
[23.644828, "o", "o"]
[23.741407, "o", "c"]
[23.959736, "o", "uments/"]
[24.683196, "o", "M"]
[25.078513, "o", "y"]
[25.292143, "o", "selfTrace/"]
[26.205617, "o", "\r\n\u001b[?2004l\r\u001b[?2004h\u001b[38;5;14m00:10:05 \u001b[38;5;224muser7\u001b[38;5;200m@\u001b[38;5;190mDubrovin02\u001b[38;5;10m$:~/Documents/MyselfTrace\r\r\n\u001b[m\u001b]12;#00ffff\u0007\u001b[38;5;14m$ \u001b[m"]
[27.127584, "o", "ins"]
[27.21712, "o", "t"]
[28.047816, "o", "\r\u001b[C\u001b[C\u001b[K"]
[28.255708, "o", "b"]
[28.417412, "o", "a"]
[28.455871, "o", "s"]
[28.666072, "o", "h"]
[28.760847, "o", " "]
[28.996508, "o", "i"]
[29.013982, "o", "n"]
[29.07282, "o", "s"]
[29.276528, "o", "t"]
[29.481236, "o", "a"]
[29.802579, "o", "ll"]
[30.181497, "o", "\r\n\u001b[?2004l\r"]
[30.383668, "o", "Defaulting to user installation because normal site-packages is not writeable\r\n"]
[30.501891, "o", "Requirement already satisfied: scapy in /home/user7/.local/lib/python3.10/site-packages (from -r requirements.txt (line 1)) (2.5.0)\r\n"]
[30.686248, "o", "\u001b[?2004h\u001b[38;5;14m00:10:09 \u001b[38;5;224muser7\u001b[38;5;200m@\u001b[38;5;190mDubrovin02\u001b[38;5;10m$:~/Documents/MyselfTrace\r\r\n\u001b[m\u001b]12;#00ffff\u0007\u001b[38;5;14m$ \u001b[m"]
[33.649299, "o", "cd"]
[33.88615, "o", "\r\n\u001b[?2004l\r\u001b[?2004h\u001b[38;5;14m00:10:13 \u001b[38;5;224muser7\u001b[38;5;200m@\u001b[38;5;190mDubrovin02\u001b[38;5;10m$:~\r\r\n\u001b[m\u001b]12;#00ffff\u0007\u001b[38;5;14m$ \u001b[m"]
[34.703154, "o", "t"]
[34.807893, "o", "r"]
[35.129371, "o", "a"]
[35.157114, "o", "c"]
[36.868251, "o", "eho"]
[37.074949, "o", "s"]
[37.844582, "o", "t"]
[38.390251, "o", "\r\n\u001b[?2004l\r"]
[38.400452, "o", "[sudo] password for user7: "]
[41.059537, "o", "\r\n"]
[41.764505, "o", "Traceback (most recent call last):\r\n  File \"/home/user7/.local/bin/tracehost\", line 118, in <module>\r\n    if args.Help or len(args) == 0:\r\nTypeError: object of type 'Namespace' has no len()\r\n"]
[41.854266, "o", "\u001b[?2004h\u001b[38;5;14m00:10:21 \u001b[38;5;224muser7\u001b[38;5;200m@\u001b[38;5;190mDubrovin02\u001b[38;5;10m$:~\r\r\n\u001b[m\u001b]12;#00ffff\u0007\u001b[38;5;14m$ \u001b[m"]
